components/animation/examples/events/index.html

```diff
@@ -58,7 +64,7 @@
       </a-entity>
     </a-scene>
 
-    <!--githubcorner-->
+    <!-- githubcorner -->
```

Comment:
Leave this as is because it's used by the site build scripts for injecting this/updating this